Client/src/index.js

```diff
@@ -26,6 +27,9 @@ function Routes() {
         <Route exact path="/">
           <HomePage />
         </Route>
+        <Route exact path="/sign-up">
+          <SignUpComponent />
```

Comment:
Not an issue but something to align on is our conventions but if we are rending from a route should it be a page/container with component vs just rendering a component? (For me i'd always think pages are rendered and components are embedded within the page - Unlike the Navigation Component that is in the Routes as that is a common component that'll run throughout the app).

Reply:
Yeah that's true, would be good to make a sign-up container that's connected to redux and we shouldn't connect the components to redux 👍